PostHog/PostHogFeatureFlags.swift

```diff
@@ -49,6 +49,32 @@ class PostHogFeatureFlags {
         }
     }
 
+    private func isRecordingActive(_ featureFlags: [String: Any], _ sessionRecording: [String: Any]) -> Bool {
+        var recordingActive = true
```

Comment:
default is active since the `sessionRecording` is a Map and not a boolean unless I find the `linkedFlag` and can compare things